src/pde_solve_ns.jl

```diff
@@ -69,13 +70,16 @@ function DiffEqBase.solve(
         mean(sum(abs2, g(X.data) - u) for (X,u) in predict_n_sde())
     end
 
+    iters = Float32[]
```

Comment:
eltype(X0)

Reply:
In pde_solve tests x0 is Array of Float64. I suppose, it should be eltype(u0(X0)[1].data) but it is always Array of Float32.

Reply:
You should make the inputs in the test be Float32 then for type stability

Reply:
ok